tests/test_worker.py

```
@@ -0,0 +1,155 @@
+from types import SimpleNamespace
+from unittest.mock import AsyncMock
+
+import pytest
+from aiosmtplib import SMTPResponseException
+from sqlalchemy.exc import OperationalError
+
+import worker
+from app.schemas.email import EmailStatus
+
+
+def _make_email(**overrides):
```

Ruff (0.14.10)

Missing return type annotation for private function `_make_email`

(ANN202)